CHANGELOG.rst

```diff
@@ -9,6 +9,10 @@ Changelog
 * Introduced support for Django 3.1
 * Dropped support for Python 2.7 and Python 3.4
 * Dropped support for Django < 2.2
+* Removed ``djangocms-column`` from the manual installation instructions
```

Comment:
@FinalAngel Should there be so many changelog entries as part of this change?

Reply:
@Aiky30 that's fine to add it here. We should ideally enforce new PRs to add changelog entries to avoid this issue